node/flows.re

```diff
@@ -412,3 +414,51 @@ let request_withdraw_proof = (state, ~hash) =>
   };
 let request_ticket_balance = (state, ~ticket, ~address) =>
   state.Node.protocol.ledger |> Ledger.balance(address, ticket);
+
+let trusted_validators_membership =
+    (
+      ~file,
+      ~persist:
+         (list(Trusted_validators_membership_change.t), ~file: string) =>
+         Lwt.t(unit),
+      state,
+      update_state,
+      request,
+    ) => {
+  open Networking.Trusted_validators_membership_change;
+  let {signature, payload: {address, action} as payload_} = request;
+  switch (
+    payload_ |> payload_to_yojson |> Yojson.Safe.to_string |> BLAKE2B.of_string
```

Comment:
`BLAKE2B.of_string` doesn't do what you want, it's not hashing it's just casting it. Probably you want to do `BLAKE2B.hash`.

Reply:
You're right. Fixing this in the next commit.